backend/routes/index.js

```diff
@@ -27,7 +30,7 @@ router.get('/messages', (req, res) => {
 // Handles POST requests to /messages
 router.post('/messages', (req, res) => {
     try {
-        Message.create(({name: req.body.name, body: req.body.body, sticker: req.body.sticker}));
+        Message.create({ name: req.body.name, body: req.body.body, sticker: req.body.sticker }, pgClient);
         res.status(200).send();
     } catch (err) {
```

Comment:
without being connected to cloudsql, trying to post produces this err:
```
(node:5336) UnhandledPromiseRejectionWarning: Error: Error: connect ENOENT /cloudsql/undefined/.s.PGSQL.5432
    at save (/ada-guestbook/backend/routes/messages.js:28:15)
(node:5336) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 4)
(node:5336) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Reply:
Fixed this. See new unit test.